tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -264,3 +266,44 @@ def test_rhcloud_external_links():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.tier3
+def test_positive_generate_all_reports_job(target_sat):
+    """Generate all reports job via foreman-rake console:
+
+    :id: a9e4bfdb-6d7c-4f8c-ae57-a81442926dd8
+
+    :Steps:
+        1. Execute Foreman GenerateAllReportsJob via foreman-rake.
+
+    :expectedresults: Reports generation works as expected.
+
+    :BZ: 2110163
+
+    :customerscenario: true
+
+    :CaseAutomation: Automated
+
+    :CaseLevel: System
```

Comment:
CaseAutomation and CaseLevel are the same in the above metadata, so this can be removed from here